src/deltachat_cursed/util.py

```diff
@@ -18,7 +18,11 @@
 from deltachat.hookspec import account_hookimpl
 
 APP_NAME = "Cursed Delta"
-APP_FOLDER = os.path.abspath(os.path.join(os.path.expanduser("~"), ".curseddelta"))
+if os.getenv("XDG_CONFIG_HOME"):
+    APP_FOLDER = os.path.abspath(os.path.join(os.path.expanduser(os.getenv("XDG_CONFIG_HOME")), "curseddelta"))
+else:
+    APP_FOLDER = os.path.abspath(os.path.join(os.path.expanduser("~"), ".curseddelta"))
```

Comment:
```suggestion
APP_FOLDER = os.path.abspath(os.path.join(os.path.expanduser(os.getenv("XDG_CONFIG_HOME", "~")), "curseddelta"))
```

Reply:
`$XDG_CONFIG_HOME` is already absolute.

The following uses `~/.config/curseddelta` if the variable isn't set:
```suggestion
APP_FOLDER = os.path.join(os.getenv("XDG_CONFIG_HOME", default=os.path.join(os.path.expanduser( '~' ), ".config")), "curseddelta")
```

Reply:
I chose to leave intact ~/.curseddelta as a way to.. honor? the original idea; like git has both ~/.git and .config/git

Reply:
why was this marked as resolved without applying my suggestion?

oth, so this variable doesn't point to home `~` then this is not the expected behavior for now

Reply:
> Another option would be using https://pypi.org/project/xdg/.

yes I think this is the way to go

Reply:
> I chose to leave intact ~/.curseddelta as a way to.. honor? the original idea; like git has both ~/.git and .config/git

If we want to be compatible with existing installations then we should check if `~/.curseddelta` exists and use `$XDG_CONFIG_HOME/curseddelta` otherwise.

Reply:
> Another option would be using https://pypi.org/project/xdg/.

actually I think I will use https://pypi.org/project/appdirs/ which handles Windows config folder as well

Reply:
> why was this marked as resolved without applying my suggestion?

that was my mistake! sorry